incubator/README.md

```diff
@@ -0,0 +1,175 @@
+
+### Start Incubation
+
+For both features and projects, the incubation process will be traced in a working group (see [Woking Group Governance](https://github.com/tikv/community/blob/master/committee/wg-governance/wg-governance.md)). Once your proposal is approved, The first order of business is to create a related working group (see [How to create a working group](https://github.com/tikv/community/blob/master/committee/wg-governance/wg-governance-zh_CN.md#创建)).
+
+In addition, each approved project will get a new GitHub repo created in [pingcap-incubator](https://github.com/pingcap-incubator) organization. Project owners are expected to add the following files to the repo:
+
+- A README for the project
+- An OWNERS from the accepted proposal
+- A CONTRIBUTING file based on[ pingcap/community](https://github.com/pingcap/community/blob/master/CONTRIBUTING.md)
+- A code-of-conduct.md based on [pingcap/community/code-of-conduct.md](https://github.com/pingcap/community/blob/master/CODE_OF_CONDUCT.md)
+- A LICENSE file with the Apache 2.0 licensea RELEASE.md file that talks about the process for releases
+
+To help you get started with these files, you can find the corresponding templates at [template project](https://github.com/pingcap-incubator/pingcap-template-project).
+
+For each incubating project, a monthly report is required to send to [pmc@pingcap.com](mailto:pmc@pingcap.com) before 10th of every month with the mentor copied.
+
+### Maturity Level Promotion
+
+When the majority of the project members determine they have met the criteria to be promoted to a higher maturity stage, they should contact their Mentors to discuss. If the Mentors agree that [the maturity criteria](./projects-approval-criteria) have been met, the project organizer can create a pull request at their quarterly meeting.
+
+PMC will arrange a promotion presentation for the project. Maturity Level Review follows [Projects Approval criteria](./projects-approval-criteria).
+
+### Graduation
+
+Projects follow the same graduation process as features, as described below:
+
+1. Create a pull request for graduation
+2. PMC members arrange a graduation presentation
+3. PMC members express their agreement by replying “LGTM (look good to me)” to the pull request
+4. Related working group disbands
+
+Graduation criteria for projects: see [Projects Approval Criteria](./projects-approval-criteria)
+
+### Retirement
+
+An incubation project will be considered retired if it meets either of the following situations:
+
+- Fails to graduate within 12 months.
+- Fails to make a release for 3 months.
+
+A warning email will be sent 3 weeks before the retirement to the Project Members and the Mentor(s). A project can be re-incubated after retirement but must go through the Incubation Process from the beginning, the same as any new project.
+
+## Benefits (WIP)
+
+## Features and Projects (WIP)
```

Comment:
I don't think it is necessary to list projects and features here, or we will modify this page frequently

Reply:
I have listed the link in the file